partner_gst_qst/views/res_partner_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="res_partner_view_form_inherit_partner_artist" model="ir.ui.view">
+        <field name="name">res.partner.view.form.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+              <xpath expr="//field[@name='vat']" position="attributes">
+                  <attribute name="invisible">1</attribute>
+             </xpath>
+             <xpath expr="//field[@name='vat']" position="after">
+                 <field name="gst" placeholder="e.g. BE0477472701"/>
+                 <field name="qst" placeholder="67890-0987JK"/>
```

Comment:
These examples of numbers are incorrect.

Here are 2 structuraly correct numbers:

GST: 123456789
QST:  1234567890TQ1234